### PR TITLE
doc: gs_testing: add rtt section

### PR DESCRIPTION
**Pull request by kapi-no**
Added a section to the sample testing part of Getting Started
documentation that explains how to use RTT for logging.

Signed-off-by: Kamil Piszczek <Kamil.Piszczek@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/gs_testing.rst [hunk 1]
```diff
@@ -22,6 +22,53 @@ Connect with the following settings:
...
+
+   .. figure:: /images/rtt_viewer_configuration.png
+      :alt: Example of RTT Viewer configuration
+
+#. Click :guilabel:`OK` to view the logging output from the device.
+
+Linux
+-----
+
+To run RTT on Linux, complete the following steps:
```
Comment (b-gent) [inline doc/nrf/gs_testing.rst:70]: @kapi-no  , this doesn't look like a complete section :)

Comment (kapi-no) [reply doc/nrf/gs_testing.rst:70]: Oops, I didn't notice leftovers from my previous approach to RTT docs. Linux section was supposed to be removed when I realized that there is no need to provide separate steps for Linux and Windows

**Comment by kapi-no** [comment]
@b-gent, I squashed your commit and removed unnecessary section. Please take another look at this PR

**Comment by kapi-no** [comment]
@carlescufi, how should I solve CODEOWNERS failure? We don't seem to have any code owner assigned to the
``doc/nrf/images``
directory? Whom should I choose?

**Comment by kapi-no** [comment]
> 
> 
> @carlescufi, how should I solve CODEOWNERS failure? We don't seem to have any code owner assigned to the
> `doc/nrf/images`
> directory? Whom should I choose?

For now, I added @ru-fu and @b-gent as ``doc/nrf`` code owners to make CI happy

**Comment by kapi-no** [comment]
CI is green. Could we merge this?

Codeowner configuration can be resolved in an independent PR if needed.

**Comment by b-gent** [comment]
> CI is green. Could we merge this?
> 
> Codeowner configuration can be resolved in an independent PR if needed.

Ok, we will figure out with @carlescufi  who should own the doc folder and resolve in a separate PR.

**Pull request merged**